### PR TITLE
feat: #1046 add test for admin management endpoints

### PR DESCRIPTION
**Pull request by MCatherine1994**

### CODE REVIEW

**Review by basilv — COMMENTED** [review]
The PR says it is adding tests, but there's a router/service/repo changes.

File: server/admin_management/api/app/repositories/user_repository.py [hunk 1]
```diff
@@ -35,6 +36,9 @@ def get_user_by_cognito_user_id(self, cognito_user_id: str) -> models.FamUser:
             .one_or_none()
         )
 
+    def get_users(self) -> List[models.FamUser]:
```
Comment (MCatherine1994) [inline server/admin_management/api/app/repositories/user_repository.py:39]: add this one for verifying the test result of adding a new user

File: server/admin_management/api/app/routers/router_application_admin.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def delete_application_admin(
     status_code=200,
     dependencies=[Depends(authorize_by_fam_admin)],
 )
-def get_application_admin_by_applicationid(
+def get_application_admin_by_application_id(
```
Comment (MCatherine1994) [inline server/admin_management/api/app/routers/router_application_admin.py:147]: rename this method just for naming consistency, forgot to change it in the last pr

Comment (basilv) [reply server/admin_management/api/app/routers/router_application_admin.py:147]: Nice!

**Review by basilv — DISMISSED** [review]

**Review by ianliuwk1019 — DISMISSED** [review]

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**